package Exercicio917;

public class MinhaExcecao extends RuntimeException {
	
	public MinhaExcecao(String _msg) {
		super(_msg);
	}
	
	public void setValor1(double _valor) {
		try {
			throw new RuntimeException();
		}
		catch(Exception e) {
			System.out.println("Erro divisão por zero");
		}		
	}
	
	public void setValor2(double _valor) throws ArithmeticException {
		throw new RuntimeException();
	}
	
}
